VCFv4.4.draft.tex

```diff
@@ -1701,6 +1706,40 @@ \subsection{Representing unspecified alleles and REF-only blocks (gVCF)}
 \end{flushleft}
 \normalsize
 
+
+\pagebreak
+\subsection{Representing copy number variation}
+\label{cnv}
+
+To encode copy number variation, VCF uses $<$CNV$>$, $<$DEL$>$ and $<$DUP$>$ symbolic structural variant alleles, CN INFO and FORMAT fields.
+
+Allele specific copy number is specified through a $<$CNV$>$ ALT allele for each distinct allelic copy number.
```

Comment:
Can `<DEL>` and `<DUP>` also be used to specify the allele specific copy number?

Reply:
DEL implies a CN<1, and DUP implies CN>2 since CN is now float, CN<1 does not necessarily mean CN=0.

That is DEL implies CICN=[0,1) and DUP CICN=(1,.] (with the slight complication that the actual CICN field is inclusive of both endpoints so the notation that I just used isn't allowed in an actual VCF).